cmd/kube-scheduler/app/config/config.go

```diff
@@ -42,12 +44,14 @@ type Config struct {
 	Authorization          apiserver.AuthorizationInfo
 	SecureServing          *apiserver.SecureServingInfo
 
-	Client          clientset.Interface
-	InformerFactory informers.SharedInformerFactory
-	PodInformer     coreinformers.PodInformer
-	EventClient     v1core.EventsGetter
-	Recorder        record.EventRecorder
-	Broadcaster     record.EventBroadcaster
+	Client                    clientset.Interface
+	InformerFactory           informers.SharedInformerFactory
+	PodInformer               coreinformers.PodInformer
+	EventClient               v1beta1.EventsGetter
+	CoreEventClient           v1core.EventsGetter
+	Recorder                  events.EventRecorder
+	Broadcaster               events.EventBroadcaster
+	LeaderElectionBroadcaster record.EventBroadcaster
```

Comment:
nit: can you group the new ones and the old ones together and add an explicit TODO to remove the old once once we fully migrate to the new api
I mean sth like:
```
EventClient ...
Record ...
Broadcaster ...

// TODO: Remove the following after fully migrating to the new events api.
CoreEventClient ...
LeaderElectionBoadcaster ...
```